services/route_manager/src/main/java/com/futurewei/alcor/route/controller/NeutronRouterController.java

```diff
@@ -302,7 +302,21 @@ public RouterInterfaceResponse addInterfaceToNeutronRouter(@PathVariable String
         if (gatewayPorts != null && gatewayPorts.size() > 0) {
             // TODO: waiting for PM new API
             // construct InternalRouterInfo
+            RouteTable neutronRouteTable = router.getNeutronRouteTable();
```

Comment:
If the router is neutron router, its NeutronRouterTable should exist because we have configured it when creating neutron router;  addInterfaceToNeutronRouter focus on neutron router, I think it is fine.

Reply:
Now it will throw an exception if NeutronRouterTable not exist

Reply:
Good. thanks.